resources/monitoring/charts/alert-rules/templates/alert-rules-rest-services.yaml

```diff
@@ -12,9 +12,9 @@ metadata:
 spec:
   groups:
   - name: rest-services.rules
-    rules:
+    rules:  ### Ref: https://prometheus.io/docs/prometheus/latest/querying/functions/#histogram_quantile
```

Comment:
please remove the comment, it is obvious that it is a regular function

Reply:
done